scss/_offcanvas.scss

```diff
@@ -41,7 +44,7 @@
       background-clip: padding-box;
       outline: 0;
       @include box-shadow(var(--#{$prefix}offcanvas-box-shadow));
-      @include transition(transform $offcanvas-transition-duration ease-in-out);
+      @include transition(transform var(--#{$prefix}offcanvas-transition-duration) ease-in-out); // Boosted mod
```

Comment:
The use of css variables here shall be told to Bootstrap so that they do it on there sitde too.

Reply:
Yep, need to see if we update on Bs too